parfiles/periodic_BC_test1.prm

```diff
@@ -0,0 +1,134 @@
+#########################################################
+# This is a variation of the platelike-boundary.prm file.
+# All settings are exactly the same with the exception
+# of the ones that pertain to compositional fields.
+# See the manual for more information about this setup.
+
+# Used to test periodic boundary conditions for a quarter
```

Comment:
put this sentence at the top of the file, remove the "rotation matrix" sentence.